app/src/main/java/com/example/thecocktailsapp/Res/CocktailsApi.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.thecocktailsapp.Rest
+
+import com.example.thecocktailsapp.model.CocktailIInfo.CocktailInfo
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface CocktailsApi:  {
+
+    @GET(cocktailsByName)
+    suspend fun getCocktailsByName(
+        @Query("s") name: String
+    ):Response<CocktailInfo>
+
+
+    @GET(cocktailsByAlcohol)
+        suspend fun getCocktailsByAlcohol(
+            @Query("a") name: String
+        ):Response<CocktailInfo>
+
+    @GET(cocktailsByRandom)
+        suspend fun getCocktailsByRandom(
+            @Query("i") name: String
+        ):Response<CocktailInfo>
```

Comment:
this wouldn't return the random cocktails, this would return a single cocktail by id, how will we be getting the random cocktails?